js/js-forms/js-forms_checkbox-input/js/index.js

```diff
@@ -11,13 +11,19 @@ function showTosError() {
   tosError.removeAttribute("hidden");
 }
 
+// make the error message hidden initially (before the form is submitted)
+hideTosError();
+
 form.addEventListener("submit", (event) => {
   event.preventDefault();
+  const tos = event.target.elements.tos.checked;
 
   // --v-- write your code here --v--
-
+  if (!tos) {
+    showTosError();
+    return;
+  }
+  hideTosError();
```

Comment:
Usually, asserting over a "positive" value is more readable than asserting over a negative one:
```suggestion
  if (tos) {
    hideTosError();
  } else {
    showTosError();
  }
```